packages/main-menu/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "@lidofinance/main-menu",
+  "version": "0.0.0-semantic-release",
+  "main": "dist/cjs/index.js",
+  "module": "dist/esm/index.js",
+  "types": "dist/esm/index.d.ts",
+  "license": "MIT",
+  "homepage": "https://github.com/lidofinance/ui",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/lidofinance/ui.git",
+    "directory": "packages/address"
+  },
+  "bugs": {
+    "url": "https://github.com/lidofinance/ui/issues"
+  },
+  "keywords": [
+    "lido",
+    "lidofinance",
+    "styled-components",
+    "component",
+    "react-component",
+    "react"
+  ],
+  "dependencies": {
+    "@lidofinance/theme": "workspace:*",
```

Comment:
Will it work specifying dependencies like this? Shouldn't we set some exact version? Not sure how it works

Reply:
this... depends on build pipeline

Reply:
@Jabher won't dependencies be installed on target machine? Or does our build pipeline update package.json?